files/cappsool/0.0.2/scripts/widget.js

```diff
@@ -0,0 +1,2749 @@
+        var lastIsElementPartiallyInViewport = wasPartiallyInViewport;
+
+        if (lastIsElementPartiallyInViewport !== newIsElementPartiallyInViewport) {
+          if (newIsElementPartiallyInViewport) {
+            //console.log("Now Partially in viewport - send event");
+            logEvent({}, eventTypes.widgetShownPartially, null, null);
+          }
+          wasPartiallyInViewport = newIsElementPartiallyInViewport;
+        }
+      }
+      if (newIsElementPartiallyInViewport != currentlyPartiallyInViewport) {
+        currentlyPartiallyInViewport = newIsElementPartiallyInViewport;
+        if (activeWidgetUi && activeWidgetUi.widgetInViewport) {
+          activeWidgetUi.widgetInViewport();
+        }
+      }
+
+      var verticalMargin = 500;
+      if (widgetConfig && widgetConfig.loadCoverMargin) {
+        verticalMargin = widgetConfig.loadCoverMargin;
+      }
+      var newIsElementAlmostInViewport = (isElementPartiallyInViewport(elem, verticalMargin) || overrideAlmost) && s;
+      if (!wasAlmostInViewport) {
+        var lastIsElementAlmostInViewport = wasAlmostInViewport;
+
+        if (lastIsElementAlmostInViewport !== newIsElementAlmostInViewport) {
+          wasAlmostInViewport = newIsElementAlmostInViewport;
+        }
+      }
+      if (newIsElementAlmostInViewport != currentlyAlmostInViewport) {
+        currentlyAlmostInViewport = newIsElementAlmostInViewport;
+        if (activeWidgetUi && activeWidgetUi.widgetInViewport) {
+          activeWidgetUi.widgetInViewport();
+        }
+      }
+
+      var newIsElementInViewport = (isElementInViewport(elem) || overrideFully) && s;
+      if (!wasFullyInViewport) {
+        var lastIsElementInViewport = wasFullyInViewport;
+
+        if (lastIsElementInViewport !== newIsElementInViewport) {
+          if (newIsElementInViewport) {
+            //console.log("Now fully in viewport - send event");
+            if (activeWidgetUi && activeWidgetUi.widgetInViewport) {
+              activeWidgetUi.widgetInViewport();
+            }
+          }
+          wasFullyInViewport = newIsElementInViewport;
+        }
+      }
+
+      if (newIsElementInViewport != currentlyInViewport) {
+        currentlyInViewport = newIsElementInViewport;
+        if (activeWidgetUi && activeWidgetUi.widgetInViewport) {
+          activeWidgetUi.widgetInViewport();
+        }
+      }
+      if (newIsElementInViewport && !wasShownEnoughTime) {
+        shownTimeCounter.start(false);
+      }
+      if (!newIsElementInViewport && !newIsElementPartiallyInViewport) {
+        shownTimeCounter.stop();
+      }
+
+    }
+    else {
+      //console.log("no widgetId")
+    }
+  }
+
+  var installCappsool = function (baseUrl) {
+
+    var log = '';
+    var a1 = document.getElementsByTagName('head')[0].getElementsByClassName("cappsool-1").length;
+    if (a1 <= 0) {
+      var script = document.createElement('script');
+      script.src = baseUrl + "/scripts/jquery-1.10.1.min.js";
```

Comment:
You use google cdns in one place to load jquery and ours in another? I suggest you use one or the other